### PR TITLE
[UA] 20210608: Mages of Strixhaven

### PR DESCRIPTION
**Pull request by NickVendel**
Base Structure by @eterean
Editing and Formatting by @NickVendel

### CODE REVIEW

File: unearthed-arcana/2021/20210608.xml [hunk 1]
```diff
@@ -0,0 +1,1397 @@
...
+			<div element="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_WORD_OF_POWER" />
+		</description>
+		<sheet display="false">
+			<description>Mages of Silverquill hone the power of words.</description>
+		</sheet>
+		<rules>
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_ELOQUENT_APPRENTICE" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_SILVERY_BARBS" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_INKY_SHROUD" level="6" />
+			<select type="Archetype Feature" name="4th Feature (Mage of Lorehold)" supports="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_INFUSION_OF_ELOQUENCE|ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_WORD_OF_POWER" level="14" />
```
Comment (Azzzzzy) [inline unearthed-arcana/2021/20210608.xml:899]: Should be "Mage of Silverquill"

File: unearthed-arcana/2021/20210608.xml [hunk 2]
```diff
@@ -0,0 +1,1397 @@
...
+	</element>
+
+	<element name="Deadly Despair" type="Archetype Feature" source="Unearthed Arcana: Mages of Strixhaven" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_WORD_OF_POWER_DEADLY_DESPAIR">
+		<compendium display="false" />
+		<sheet>
+			<description>When the target of your Silvery Barbs fails an attack roll, an ability check, or a saving throw because of the reroll, you can invoke a word of despair to give the target vulnerability to one damage type of your choice until the start of your next turn.</description>
+		</sheet>
+	</element>
+
+	<element name="Selfless Invocation" type="Archetype Feature" source="Unearthed Arcana: Mages of Strixhaven" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_WORD_OF_POWER_DEADLY_DESPAIR">
```
Comment (Azzzzzy) [inline unearthed-arcana/2021/20210608.xml:1202]: Wrong ID

File: unearthed-arcana/2021/20210608.xml [hunk 3]
```diff
@@ -0,0 +1,1397 @@
...
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_WITHERBLOOM_WITHERING_VORTEX" level="14" />
+		</rules>
+	</element>
+
+	<element name="Witherbloom Spells" type="Archetype Feature" source="Unearthed Arcana: Mages of Strixhaven" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_WITHERBLOOM_WITHERBLOOM_SPELLS_DRUID">
+		<description>
+			<p><em>Level 1+ Mage of Witherbloom Feature</em></p>
+			<p>You learn the cantrip <i>spare the dying</i> and the 1st-level spells <i>cure wounds</i> and <i>inflict wounds</i>. You learn additional spells when you reach certain levels in this class, as shown on the Witherbloom Spells table.</p>
+			<p class="indent">Each of these spells count as a class spell for you, but it doesn’t count against the number of spells you know. If you are a druid, you always have the spells prepared, and they don’t count against the number of spells you can prepare each day.</p>
+			<h5>Lorehold Spells</h5>
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210608.xml:1244]:
```suggestion
			<h5>Witherbloom Spells</h5>
```

File: unearthed-arcana/2021/20210608.xml [hunk 4]
```diff
@@ -0,0 +1,1397 @@
...
+			</div>
+		</description>
+		<sheet display="false" />
+		<spellcasting name="Warlock" extend="true">
+			<extend>ID_PHB_SPELL_DARKNESS</extend>
+		</spellcasting>
+		<rules>
+			<select type="Spell" name="Spell (Inky Shroud)" supports="$(spellcasting:list), $(spellcasting:slots)" default="ID_PHB_SPELL_DARKNESS" spellcasting="Warlock" />
+		</rules>
+	</element>
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210608.xml:1039]: These spells should just be granted for the appropriate spellcating ability.  Nothing in Eloquent Apprentice or Inky Shroud implies that they can be swapped later.  Other subclasses like the Divine Soul and the TCoE sorcerers have specific language for swapping the spells.

**Review by kgirtxd — CHANGES_REQUESTED** [review]

**Review by kgirtxd — APPROVED** [review]

**Review by Azzzzzy — APPROVED** [review]

**Pull request merged**